Source/WebCore/rendering/svg/RenderSVGResourceSolidColor.cpp

```diff
@@ -50,7 +50,9 @@ bool RenderSVGResourceSolidColor::applyResource(RenderElement& renderer, const R
         else
             context->setAlpha(1);
         context->setFillColor(style.colorByApplyingColorFilter(m_color));
-        if (!isRenderingMask)
+        if (isRenderingMask)
```

Comment:
Ok. How does it work for legacy?

Reply:
It seems to have no effect there. I can add LBSE ifdef and runtime flag check if you want, looks like it is not needed though.

Reply:
I rather meant to ask why this is not necessary for legacy?